### ISSUE TITLE
Trouble Installing

### ISSUE THREAD

**Issue by Thak676** [recp/cglm#38]
I am trying to install on a different machine now (mac). When I run sh autogen.sh, I get the following:

autogen.sh: line 20: autoheader: command not found
autogen.sh: line 21: libtoolize: command not found
autogen.sh: line 22: aclocal: command not found
autogen.sh: line 23: autoconf: command not found
autogen.sh: line 24: automake: command not found

**Comment by Thak676** [comment]
Fixed.

I did brew install autoconf automake libtool
and then it worked. Sorry if this was obvious.

**Comment by recp** [comment]
Yes, **autotools** (automake, autoconf, libtool...) must be installed before run build scripts for autotools. Maybe I should update README about how to install autotools
